wait-for-deployment

```diff
@@ -36,11 +41,22 @@ get_deployment_jsonpath() {
   kubectl get deployment "${deployment}" -o "jsonpath=${_jsonpath}"
 }
 
-if [[ $# != 1 ]]; then
-  echo "usage: $(basename $0) <deployment>" >&2
+if [[ $# -lt 1 ]]; then
+  echo "usage: $(basename $0) <deployment> [timeout]" >&2
```

Comment:
Should say `[<timeout>]`.

Can you add two extra lines explaining `<deployment>` and `<timeout>`, respectively? The description on `<timeout>` should mention that the value must be in seconds and that omitting means no timeout will be applied.